google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -74,6 +74,9 @@
  *   <li>A "callable" method, like `readRowsCallable()`. This type of method takes no parameters and
  *       returns an immutable API callable object, which can be used to initiate calls to the
  *       service.
+ *   <li>Each read or write operation in this client is built around the callables in the form of
+ *       synchronized or asynchronized operation. These operation can also be achieved by directly
+ *       using callables.
```

Comment:
Please provide an example

Reply:
Actually looking at the docs, it seems like the callables info is already there:

```
*   <li>A "callable" method, like `readRowsCallable()`. This type of method takes no parameters and
 *       returns an immutable API callable object, which can be used to initiate calls to the	 *       returns an immutable API callable object, which can be used to initiate calls to the
 *       service.	 *       service.
```

I think all this needs is just an example:

Taking ReadRows as an example:

```java
// These two invocation are equivalent
ServerStream<Row> stream1 = client.readRows(query);
ServerStream<Row> stream2 = client.readRowsCallable().call(query);

// These two invocation are also equivalent
client.readRowsAsync(query, observer);
client.readRowsCallable().call(query, observer); 
```